src/ApplicationInsights.OwinExtensions/Consts.cs

```diff
@@ -3,5 +3,6 @@
     public class Consts
     {
         public const string OperationIdContextKey = "ApplicationInsights.OwinExtensions.OperationIdContext";
+        public const string OperationIdHeaderName = "AI-Operation-Id";
```

Comment:
This should conform to convention for naming custom headers, which is X- ... I suggest using X-Operation-Id.